src/main/java/edu/kis/powp/jobs2d/drivers/adapter/LineDrawerAdapter.java

```diff
@@ -0,0 +1,26 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+
+public class LineDrawerAdapter extends DrawerAdapter {
+    private int startX = 0, startY = 0;
+    private ILine chosenLine;
+    private DrawPanelController controller;
```

Comment:
fields already existing in DrawPanelController are duplicated. Is DrawerAdapter implementation is actually re-used or only the interface?